### ISSUE TITLE
OS Error: SavedModel file does not exist

### ISSUE THREAD

**Issue by jae0224** [tanjeffreyz02/auto-maple#46]
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\threading.py", line 973, in _bootstrap_inner
    self.run()
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\Jae\Downloads\auto-maple-2\auto-maple-2\bot.py", line 64, in _main
    model = detection.load_model()
  File "C:\Users\Jae\Downloads\auto-maple-2\auto-maple-2\detection.py", line 19, in load_model
    return tf.saved_model.load(model_dir)
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\site-packages\tensorflow\python\saved_model\load.py", line 936, in load
    result = load_internal(export_dir, tags, options)["root"]
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\site-packages\tensorflow\python\saved_model\load.py", line 949, in load_internal
    loader_impl.parse_saved_model_with_debug_info(export_dir))
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\site-packages\tensorflow\python\saved_model\loader_impl.py", line 57, in parse_saved_model_with_debug_info
    saved_model = parse_saved_model(export_dir)
  File "C:\Users\Jae\AppData\Local\Programs\Python\Python39\lib\site-packages\tensorflow\python\saved_model\loader_impl.py", line 115, in parse_saved_model
    raise IOError(
OSError: SavedModel file does not exist at: assets/models/rune_model_rnn_filtered_cannied/saved_model\{saved_model.pbtxt|saved_model.pb}

I'm getting this when I run main.py, i've installed all the requirements for the program. If you could help with fixing this that would be great.

**Comment by tanjeffreyz02** [comment]
Hi, have you finished all the [setup instructions](https://github.com/tanjeffreyz02/auto-maple#--setup) as well? Please let me know if that helps.

**Comment by jae0224** [comment]
ohhh omg thats the one part of setup I looked past because I saw tensorflow and I was already using your program from before you did all these updates haha. Sorry about that. 

I'm getting this message for runes now and i'm assuming I don't have enough memory? 

Solving rune:
2022-03-09 21:16:25.217158: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Loaded cuDNN version 8100
2022-03-09 21:16:27.130321: W tensorflow/core/common_runtime/bfc_allocator.cc:275] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.32GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2022-03-09 21:16:27.196821: W tensorflow/core/common_runtime/bfc_allocator.cc:275] Allocator (GPU_0_bfc) ran out of memory trying to allocate 2.60GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
2022-03-09 21:16:27.285303: W tensorflow/core/common_runtime/bfc_allocator.cc:275] Allocator (GPU_0_bfc) ran out of memory trying to allocate 3.59GiB with freed_by_count=0. The caller indicates that this is not a failure, but may mean that there could be performance gains if more memory were available.
